### PR TITLE
updates to data strategies

### PR DESCRIPTION
**Pull request by setheliot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,11 @@
...
+| \- High availability: Ensures data is replicated and available. | \- Design a data replication strategy.<br>\- Implement a highly available database system.                 | **Data Replication**             | Ensures data durability, to ensure system availability.                   | Increases costs, consistency issues.                | \- Use asynchronous replication.<br>\- Implement conflict resolution.<br>\- Use multi-master replication.                                          | AWS RDS standby (synchronous), AWS RDS Read Replicas (asynchronous), MongoDB Replica Set (asynchronous)|
+| \- Disaster recovery: Provides data protection.                 | \- Design a data backup strategy.<br>\- Implement a disaster recovery plan.                 | **Data Backup**             | Ensures data recoverability.                   | Increases costs, if backups are not up to date there will be data loss, backup may not be accessible.                | \- Use continuous or automated backup.<br>\- Backup to another region. resolution.                                          | Use native backup capabilites of the data store, or centralized backup products like AWS Backup, Veeam, or ARPIO|
```
Comment (bhavul) [inline README.md]: Hey this one is already present as "Backup and Recovery" component. So could you remove this "Disaster recovery" row? 
Rest additions/edits look good to me. :)

**Review by bhavul — CHANGES_REQUESTED** [review]
Thanks for making this better. Just one duplication which I'd like you to remove and then will merge this.

File: README.md (hunk 1 above)
Comment (setheliot) [reply README.md]: ok, done!

**Review by bhavul — APPROVED** [review]

**Comment by bhavul** [comment]
Thanks for your contributions! Appreciate it. :)

**Pull request merged**